### ISSUE TITLE
Unable to unselect a "grouping" or "category" taxonomy

### ISSUE THREAD

**Issue by technicallyerik** [bolt/core#1214]
I suppose I don't know if this a feature or a bug, but I am unable to unselect a "grouping" or "category" taxonomy option.  

Details
-------

| Question                      | Answer
|----------------- | ------
| Relevant Bolt Version | 4.0.0-rc.7
| Install type                   |  Composer install
| PHP version                 | 7.2.22
| Web server                  | Apache
| For UX/UI issues        | Safari 13.0.5

Reproduction
------------

### Specifics

I setup my taxonomy as follows:

```
categories:
    name: Categories
    slug: categories
    singular_name: Category
    singular_slug: category
    behaves_like: categories
    multiple: false
    options: [ Category 1, Category 2, Category 3 ]

series:
    name: Series
    slug: series
    singular_name: Series
    singular_slug: Series
    behaves_like: grouping
    options: [ Series 1, Series 2, Series 3 ]
```

### Expected result

I am able to not select an option when creating an entry using these taxonomy.  I would expect the dropdown to default to empty ("Select Option") similar to how tags function.

### Actual result

New entries are defaulted to the first taxonomy option, with no option to select none:

<img width="739" alt="Screen Shot 2020-03-22 at 10 24 29 PM" src="https://user-images.githubusercontent.com/2567597/77277914-03037180-6c8c-11ea-8544-c40fe44b1590.png">

**Comment by JTNMW** [comment]
The field config option in contenttypes.yaml 'allowempty: true' allows the select field to be set to empty. 

However, I also would like to have the default set to empty. At the moment all new records have the first taxonomy automatically.

**Comment by technicallyerik** [comment]
@JTNMW - To clarify, I'm not using a select field on my content type.  The select fields in my screenshot above are from my taxonomy:
`taxonomy: [ categories, tags, series ]`

I attempted to add `allowempty: true` to my taxonomy definition but it didn't appear to have an effect.

**Comment by bobdenotter** [comment]
Yup, this is buggy! We'll get this fixed. :-)

**Comment by bobdenotter** [comment]
Thanks for reporting! This is fixed in #1232
